pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -115,8 +115,12 @@ func (r *REST) Update(ctx kapi.Context, tagName string, objInfo rest.UpdatedObje
 			return nil, false, kapierrors.NewBadRequest("namespace is required on ImageStreamTags")
 		}
 		imageStream = &api.ImageStream{
-			ObjectMeta: kapi.ObjectMeta{Namespace: namespace, Name: name},
+			ObjectMeta: kapi.ObjectMeta{
+				Namespace: namespace,
+				Name:      name,
+			},
 		}
+		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
```

Comment:
We already supported this for tags (PUT tag is create on update).

On Wed, Jul 6, 2016 at 10:15 AM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/image/registry/imagestreamtag/rest.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69738065>:
>
> >  		}
> > +		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
>
> I don't know what this is doing... not sure we have createonupdate for
> other virtual resources
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/993c4ac1fd086af40ae0e0c811ce9052ce185714..be6ceebe60f75425e3317b336ed15b7354e9f4b1#r69738065>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p4O1bXWoSzpj6QoFEfUEeLvWsZp7ks5qS7hugaJpZM4I-xTE>
> .
>